docs/core/versions/binding.md

```diff
@@ -0,0 +1,117 @@
+
+The following example shows the `global.json` syntax:
+
+``` json
+{
+  "sdk": {
+    "version": "2.0.0"
+  }
+}
+```
+
+The process for selecting an SDK version is:
+
+- `dotnet` searches for a `global.json` file iteratively reverse-navigating the path upward from the current working directory.
+- `dotnet` uses the SDK specified in the first `global.json` found.
+- `dotnet` binds to the latest installed SDK if no `global.json` is found.
+
+## Target Framework Monikers define build time APIs
+
+You build your project against APIs defined in a **Target Framework Moniker** (TFM). You specify the target framework in the project file. Set the `TargetFramework` element in your project file as shown in the following example:
+
+``` xml
+<TargetFramework>netcoreapp2.0</TargetFramework>
+```
+
+You may build your project against multiple TFMs. Setting multiple target frameworks is more common for libraries but can be done with applications as well. You specify a `TargetFrameworks` property (plural of `TargetFramework`). The target frameworks will be semicolon-delimited as shown in the following example:
+
+``` xml
+<TargetFrameworks>netcoreapp2.0;net47</TargetFrameworks>
+```
+
+A given SDK supports a fixed set of frameworks, capped to the target framework of the runtime it ships with. For example, the .NET Core 2.0 SDK includes the .NET Core 2.0 runtime, which is an implementation of the `netcoreapp2.0` target framework. The .NET Core 2.0 SDK will support `netcoreapp1.0`, `netcoreapp1.1`, and `netcoreapp2.0` but not `netcoreapp2.1` (or higher). You install the .NET Core 2.1 SDK to build for `netcoreapp2.1`
+
+.NET Standard target frameworks are also capped in the same way. The .NET Core 2.0 SDK is capped to `netstandard2.0`.
+
+You can override the minimum runtime patch version (to higher or lower versions) in the project file, as shown in the following example:
+
+``` xml
+<RuntimeFrameworkVersion>2.0.4</RuntimeFrameworkVersion>
+```
+
+ The `RuntimeFrameworkVersion` element  overrides the default version policy. When you target multiple frameworks, you must specify the minimum runtime patch version to the specific target framework, as shown in the following example for .NET Core:
+
+``` xml
+<TargetFrameworks>netcoreapp2.0;net47</TargetFrameworks>
+<RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.4</RuntimeFrameworkVersion>
+```
+
+## dotnet run rolls forward
```

Comment:
I think this section should probably not be specific to `dotnet run`.  It should be for running a framework-dependent app, which can be via `dotnet run` or via `dotnet app.dll`.  In the .NET Core SDK 2.1.400, [we will support](https://github.com/dotnet/cli/issues/6237#issuecomment-398181320) framework-dependent deployments which include an executable which can be launched natively.  So for Windows you will be able to run them by launching `app.exe`, and on other OS's, you would just run `app`.

Reply:
I changed the header to use the term "Framework-dependent apps" Is that what we want to standardize on?